pkg/providers/amifamily/ami.go

```diff
@@ -219,11 +219,22 @@ func (p *Provider) fetchAMIsFromEC2(ctx context.Context, amiSelector map[string]
 	p.ec2Cache.SetDefault(fmt.Sprint(hash), output.Images)
 	amiIDs := lo.Map(output.Images, func(ami *ec2.Image, _ int) string { return *ami.ImageId })
 	if p.cm.HasChanged("amiIDs", amiIDs) {
-		logging.FromContext(ctx).With("ami-ids", amiIDs).Debugf("discovered images")
+		logging.FromContext(ctx).With("ami-ids", concatenateAmiIDs(amiIDs)).Debugf("discovered images")
```

Comment:
```suggestion
		logging.FromContext(ctx).With("ids", concatenateAmiIDs(amiIDs), "count", len(amiIDs)).Debugf("discovered images")
```

We've already scoped this to "images" so I think just saying "ids" is fine. Also, I think making "count" a structured value is probably useful here as well